app/controllers/offer/search_label.js

```diff
@@ -0,0 +1,72 @@
+import searchLabelController from "../search_label";
+
+export default searchLabelController.extend({
+  // ----- Arguments -----
+  packageType: null,
+  item: null,
+
+  //--Ajax Methods
+  createItem() {
+    const item = this.get("store").createRecord("item", this.itemParams());
+    this.runTask(
+      item.save().then(itemResponse => {
+        this.set("item", itemResponse);
+        this.addPackage();
+      })
+    );
+  },
+
+  addPackage() {
+    let pkgRecord = this.store.createRecord("package", this.packageParams());
+    this.runTask(
+      pkgRecord.save().then(pkg => {
+        this.transitionToRoute("receive_package", pkg.id, {
+          queryParams: { isUnplannedPackage: true }
+        });
+      })
+    );
+  },
+
+  //--Params helpers
+  itemParams() {
+    const packageType = this.get("packageType");
+    const offer = this.get("model");
+    const defaultDonorCondition = this.get("store")
+      .peekAll("donorCondition")
+      .filterBy("name", "Lightly Used")
+      .get("firstObject");
```

Comment:
This seems to be a reusable value `defaultDonorCondition` 

(also defined at https://github.com/crossroads/admin.goodcity/blob/a7213ced895e730473d880a5f44311696a62c188/app/controllers/review_offer/receive.js#L12-L15)

Can we define it some other common place and then reuse it?

Reply:
I'm more concerned with the string comparison to `"Lightly Used"` than the code repetition. That's a time bomb, we never want to compare strings. e.g case: Matt could one day decide to change the label and this would break

Reply:
ah right.. I think even it would break if Admin switches to Chinese language.

Reply:
Handled this situation with `||` condition.  Need to add `default` field on `donor_condition` table.